File: AtCoder/abc210/A.swift
private func readLn() -> String {
	return readLine() ?? ""
}

private func readLns() -> [String] {
	return readLn().split(separator: " ").map{ String($0) }
}

private func readInts() -> [Int] {
	return readLns().compactMap{ Int($0) }
}

let NAXY = readInts()
if NAXY[0] <= NAXY[1] {
	print(NAXY[0] * NAXY[2])
} else {
	print(NAXY[1] * NAXY[2] + (NAXY[0] - NAXY[1]) * NAXY[3])
}

File: AtCoder/abc210/A.kt
private fun readLn() = readLine() ?: ""
private fun readLns() = readLn().split(" ")
private fun readInts() = readLns().map { it.toInt() }

fun main() {
    val NAXY = readInts()
    if (NAXY[0] <= NAXY[1]) {
        print(NAXY[0] * NAXY[2])
    } else {
        print(NAXY[1] * NAXY[2] + (NAXY[0] - NAXY[1]) * NAXY[3])
    }
}